libdnf/dnf-repo.cpp

```diff
@@ -76,14 +73,12 @@ typedef struct
     gint64           timestamp_modified;    /* µs */
     GError          *last_check_error;
     GKeyFile        *keyfile;
-    GHashTable      *filenames_md;          /* md filename (e.g. "primary") -> file path */
     DnfContext      *context;               /* weak reference */
     DnfRepoKind      kind;
-    HyRepo           repo;
+    libdnf::Repo    *repo;
```

Comment:
Why you changed type, but see bellow (I know that both type are identical):
    if (priv->repo != NULL)
        hy_repo_free(priv->repo);